myhpom/tests/test_user_model.py

```diff
@@ -42,4 +39,11 @@ def test_save_invalid_user_fails(self):
         user_data.update(first_name='A', last_name='b', email='Ab@')
         user = User(**user_data)
         self.assertRaises(ValidationError, user.save)
-                
+
+    def test_get_full_name(self):
+        """User.get_full_name() includes first, middle, last name"""
+        user = UserFactory()  # first_name, userdetails.middle_name, last_name all here & normalized
+        self.assertEqual(
+            user.get_full_name(),
+            ' '.join([user.first_name, user.userdetails.middle_name, user.last_name]),
+        )
```

Comment:
This assertion is a duplication of the implementation under test - An explicit 'when, then' pattern would clearly show the expected behavior (ie, `user = Userfactory(first_name='first',last_name=''); self.assertEqual('first', user.get_full_name())`)